Packet++/src/GtpLayer.cpp

```diff
@@ -582,7 +581,7 @@ namespace pcpp
 
 		// GTP-U message, try to parse the next layer
 
-		uint8_t* payload = (uint8_t*)(m_Data + headerLen);
+		auto payload = (uint8_t*)(m_Data + headerLen);
```

Comment:
Isn't it better `auto *` instead of `auto`. clang-tidy will warn about it https://clang.llvm.org/extra/clang-tidy/checks/readability/qualified-auto.html

Reply:
Fixed in https://github.com/seladb/PcapPlusPlus/pull/1634/commits/bca9a0e95c65db04920aba928cc11303a9f66730